### PR TITLE
Remove the Justice Department

### PR DESCRIPTION
**Pull request by Elijahrane**
# Description

Removes the justice department.

# Changelog

:cl: Rane
- remove: Removed the Justice Department.

### CODE REVIEW

**Comment by ShatteredSwords** [comment]
The whole point of porting Justice was so EE could port stuff from Delta without any issues 🙃

**Comment by gluesniffler** [comment]
based

File: Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 1]
```diff
@@ -244,50 +244,6 @@
...
-- type: entity
-  parent: ClothingUniformBase
-  id: ClothingUniformJumpsuitChiefJusticeFormal
-  name: chief justice's formal jumpsuit
-  description: A fancy double-breasted suit with golden accoutrements. Sharp and authoritative.
-  components:
-  - type: Sprite
-    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/cjformal.rsi
-  - type: Clothing
-    sprite: DeltaV/Clothing/Uniforms/Jumpsuit/cjformal.rsi
```
Comment (VMSolidus) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml:267]: Leave stuff like clothing item assets in, please. They can just be added as loadout items for Lawyers to take.

File: Resources/Prototypes/DeltaV/Entities/Clothing/Head/hats.yml [hunk 2]
```diff
@@ -191,16 +191,4 @@
   - type: Sprite
     sprite: DeltaV/Clothing/Head/Hats/beret_corpsman.rsi
   - type: Clothing
-    sprite: DeltaV/Clothing/Head/Hats/beret_corpsman.rsi
-
-- type: entity
```
Comment (VMSolidus) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Head/hats.yml:196]: Same with hats, cloaks, etc.

**Review by VMSolidus — CHANGES_REQUESTED** [review]

File: Resources/Migrations/eeMigration.yml [hunk 3]
```diff
@@ -4,4 +4,57 @@ CrateFunLizardPlushieBulk: CrateFunToyBox
...
+LockerclerkFilled: null
+ClerkPDA: null
+ClerkIDCard: null
+RubberStampNotary: null
+EncryptionKeyJustice: null
+ClothingHeadsetJustice: null
+DoorRemoteJustice: null
+ClothingHeadHatCJToque: null
+SignDirectionalJuistice: null
+PaperStationWarrant: null
```
Comment (VMSolidus) [inline Resources/Migrations/eeMigration.yml:60]:
```suggestion
ChiefJusticePDA: null
ChiefJusticeIDCard: null
LockerChiefJustice: null
LockerChiefJusticeFilled: null
SpawnPointChiefJustice: null
DefaultStationBeaconJustice: null
DefaultStationBeaconChiefJustice: null
DefaultStationBeaconProsecutor: null
RubberStampChiefJustice: null
AirlockJustice: Airlock
AirlockJusticeGlass: AirlockGlass
AirlockChiefJusticeLocked: AirlockLawyerLocked
AirlockProsecutorLocked: AirlockLawyerLocked
AirlockChiefJusticeGlassLocked: AirlockLawyerGlassLocked
AirlockProsecutorGlassLocked: AirlockLawyerGlassLocked
AirlockJusticeLocked: AirlockLawyerLocked
AirlockMaintChiefJusticeLocked: AirlockLawyerLocked
AirlockMaintJusticeLocked: AirlockLawyerLocked
AirlockMaintProsecutorLocked: AirlockLawyerLocked
WindoorSecureChiefJusticeLocked: WindoorSecureLawyerLocked
WindoorSecureProsecutorLocked: WindoorSecureLawyerLocked
WindoorSecureJusticeLocked: WindoorSecureLawyerLocked
DoorElectronicsChiefJustice: DoorElectronicsLawyer
DoorElectronicsJustice: DoorElectronicsLawyer
DoorElectronicsProsecutor: DoorElectronicsLawyer
ProsecutorPDA: null
BoxPDAJustice: null
SpawnPointProsecutor: null
SpawnPointClerk: null
ProsecutorIDCard: null
LockerClerk: null
LockerclerkFilled: null
ClerkPDA: null
ClerkIDCard: null
RubberStampNotary: null
EncryptionKeyJustice: null
ClothingHeadsetJustice: null
DoorRemoteJustice: null
SignDirectionalJuistice: null
PaperStationWarrant: null
```
Okay, I've put in a list of all the items I'd like for us to keep, but to instead just be Lawyer Loadout Items.

**Review by VMSolidus — CHANGES_REQUESTED** [review]

**Comment by Flybik** [comment]
I then propose to immediately remove all roles except engineers and force them to build the station during their shift. 😡

**Comment by FoxxoTrystan** [comment]
Is this PR still getting worked on?

**Comment by FoxxoTrystan** [comment]
> Presumably I'm supposed to be repobanned so I never bothered to check

Huh? wha?
@VMSolidus ?

**Comment by VMSolidus** [comment]
> Presumably I'm supposed to be repobanned so I never bothered to check

I literally don't have the ability to repoban people. I also don't care to repoban people unless they're doing something ridiculous like making a PR that replaces every word in the entire repo with a racial slur.

**Comment by Elijahrane** [comment]
Well I'm not really interested in finishing this PR but I may have something to contribute at a later date, who knows

**Pull request closed without merging**